cmd/get-measurements/main.go

```diff
@@ -0,0 +1,164 @@
+package main
+
+//
+// Make a HTTP GET request over a TEE-attested connection (to a server with aTLS support),
+// and print the verified measurements and the response payload.
```

Comment:
Does this CLI only fetch and save's the measurements or verifies them too?
If I understood the implementation down correctly, it is just extracting them from the header and storing them. Where does the verification/attestation of those measurements done before storing them?

Reply:
it verifies them first. that's part of the TLS handshake that it does just like cvm-proxy-client here: https://github.com/flashbots/cvm-reverse-proxy/blob/99238f6abe5b8107a81c0a0aebadb57316154c9f/cmd/get-measurements/main.go#L127-L138